Source/DataSources/networkLinkUpdateMgr.js

```diff
@@ -0,0 +1,91 @@
+export default function networkLinkUpdateMgr() {
```

Comment:
This isn't how we generally define classes in Cesium. This is just a function that returns a anonymous object.  We usually do something like this:
```
function NetworkLinkUpdateMgr() {
  this._kmlSources = [];
}
NetworkLinkUpdateMgr.prototype.addLink = function(href, kml, dataSource) {
  ...
}
NetworkLinkUpdateMgr.prototype.processUpdate = function(updateNode) {
  ...
}
```

Reply:
Will do 👍